apps/web/src/components/Layout/MaintenanceBanner.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import isAfter from "date-fns/isAfter";
+import isBefore from "date-fns/isBefore";
+import isEqual from "date-fns/isEqual";
+
+import {
+  currentDate,
+  parseDateTimeUtc,
+  relativeClosingDate,
+} from "@gc-digital-talent/date-helpers";
+import { getRuntimeVariable } from "@gc-digital-talent/env";
+import { Alert } from "@gc-digital-talent/ui";
+
+const MaintenanceBanner = () => {
+  const intl = useIntl();
+  // Outage time range, date, starttime
+  const maintenanceBannerPublicDate = getRuntimeVariable(
+    "MAINTENANCE_BANNER_PUBLIC_DATE",
+  );
+  const serverMaintenanceDate = getRuntimeVariable(
+    "MAINTENANCE_BANNER_START_DATE",
+  );
+  const serverMaintenanceDuration = getRuntimeVariable(
+    "MAINTENANCE_BANNER_DURATION",
+  );
+
+  console.log(serverMaintenanceDate);
+  const showMaintenanceBanner =
+    maintenanceBannerPublicDate && serverMaintenanceDate
+      ? (isEqual(
+          parseDateTimeUtc(maintenanceBannerPublicDate),
+          parseDateTimeUtc(currentDate()),
```

Comment:
This is a little odd, to turn the current date to an ISO UTC string then parse it back to a Date.  You can just do new Date() to get the current date as a Date object.

Reply:
Ya good point 😆 Fixed by [88f4c06](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7832/commits/88f4c06c3867f6804a4f87b8a770c5ce0b996bfd)